### PR TITLE
Fix siScale imprecision errors

### PR DESCRIPTION
**Pull request by alfonnews**
Fix siScale imprecision errors

### CODE REVIEW

File: pyqtgraph/functions.py [hunk 1]
```diff
@@ -75,9 +75,8 @@ def siScale(x, minVal=1e-25, allowUnicode=True):
             pref = SI_PREFIXES[m+8]
         else:
             pref = SI_PREFIXES_ASCII[m+8]
-    p = .001**m
-    
-    return (p, pref)
```
Comment (2xB) [inline pyqtgraph/functions.py]: The `return` statement being removed looks like a mistake?

Comment (2xB) [reply pyqtgraph/functions.py]: To fix this, just commit the fix to your branch `alfonnews:Fix_siScale_imprecision_errors`. Thank you!

File: pyqtgraph/functions.py [hunk 2]
```diff
@@ -75,9 +75,8 @@ def siScale(x, minVal=1e-25, allowUnicode=True):
             pref = SI_PREFIXES[m+8]
         else:
             pref = SI_PREFIXES_ASCII[m+8]
-    p = .001**m
-    
-    return (p, pref)    
+    m1 = -3*m
+    p = float('1E{0}'.format(m1))
```
Comment (2xB) [inline pyqtgraph/functions.py]: Somewhat cleaner is
```suggestion
    p = 10.**m1
```
, as it abstains from any type casting and formatting, while ensuring the same amount of precision.

**Comment by j9ac9k** [comment]
Grr...how did that other commit get in here...

**Pull request merged**